llama_stack/apis/agents/agents.py

```diff
@@ -604,6 +604,7 @@ async def create_openai_response(
         stream: bool | None = False,
         temperature: float | None = None,
         tools: list[OpenAIResponseInputTool] | None = None,
+        max_infer_iters: int | None = 10,  # this is an extension to the OpenAI API
```

Comment:
@ehhuang @bbrowning I am going ahead with this for now, but let me know if you have thoughts on how we should do these extensions. If there is a well accepted pattern for doing this, we can adopt that.